configure.ac

```diff
@@ -115,6 +119,24 @@ AC_SUBST(bootdir)
 AC_DEFINE_UNQUOTED(BOOT_DIRECTORY, "$bootdir",
                    [The normalised path to the boot/ESP mount])
 
+# The vendor name to be used in emitted boot entries
+AC_ARG_WITH([os-name], AS_HELP_STRING([--with-os-name],
+        [the OS name]), [osname=${withval}],
+        [osname="Clear Linux Software for Intel Architecture"])
```

Comment:
Hrm this is a nice change for backwards compatibility. At the same time what do you think about using a generic-linux-os type name as the default?